squad/api/rest.py

```diff
@@ -199,7 +199,7 @@ class TestFilter(filters.FilterSet):
 
     class Meta:
         model = Test
-        fields = {'name': ['exact', 'in', 'startswith', 'contains', 'icontains'],
+        fields = {'metadata__name': ['exact', 'in', 'startswith', 'contains', 'icontains'],
```

Comment:
does this break existing queries that are using `name`? is is possible to have `name`  as an alias to `metadata__name`?

Reply:
It did break yeah, I looked it up in `django_filters` and made this work
```python
name = filters.CharFilter(lookup_expr='icontains', field_name='metadata__name')
```
I think `icontains` will cover `['exact', 'in', 'startswith', 'contains', 'icontains']` and we'll still preserve name queries like `/api/tests/?name=test_receives_metadata_file`